extensions/wikia/WikiaStyleGuide/css/tooltip-icon.scss

```diff
@@ -0,0 +1,24 @@
+@import 'skins/shared/color';
+
+.tooltip-icon-wrapper {
+	align-items: center;
+	display: inline-flex;
+}
+
+%tooltip-icon {
+	background-color: $color-buttons;
+	border-radius: 6px;
+	color: white;
+	cursor: default;
+	display: inline-block;
+	font-size: 10px;
+	height: 12px;
+	line-height: 12px;
+	margin: 0 3px;
+	text-align: center;
+	width: 12px;
+}
+
+.tooltip-icon {
+	@extend %tooltip-icon;
```

Comment:
It is a bit of a pre-mature optimization, since nowhere else actually uses that placeholder at this point. It should probably just be removed.